resources/package.json

```diff
@@ -4,35 +4,40 @@
   "private": true,
   "homepage": "./",
   "dependencies": {
-    "@craco/craco": "^6.1.1",
+    "@geist-ui/react": "^2.1.5",
+    "@geist-ui/react-icons": "^1.0.1",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
+    "@testing-library/user-event": "^13.1.8",
     "@types/jest": "^26.0.20",
-    "@types/node": "^14.14.34",
+    "@types/node": "^15.0.2",
     "@types/react": "^17.0.3",
```

Comment:
you can move the `@types/**` packages to `devDependencies` because it's only useful for developers using an IDE. Not for build or run tests.

Reply:
It's required for TypeScript I think — not _just_ IDEs.

Reply:
Yes those `@types` are also required for just building the application. If you're creating a new application using [Create React App](https://create-react-app.dev/) those packages are placed in the `dependencies` section. 

My proposal would be to keep all dependencies except `react-git-info` (move to `dependencies`) in place.